website/docs/r/linux_virtual_machine.html.markdown

```diff
@@ -140,6 +140,8 @@ The following arguments are supported:
 
 -> **NOTE:** When an `admin_password` is specified `disable_password_authentication` must be set to `false`.
 
+* `encryption_at_host_enabled` - (Optional) Should Azure encrypt the data stored on the VM host at rest and flows encrypted to the Storage service.
```

Comment:
The Azure API uses slightly different terminology for this:

> This will enable the encryption for all the disks including Resource/Temp disk at host itself

could we make this:

```suggestion
* `encryption_at_host_enabled` - (Optional) Should all of the disks (including the temp disk) attached to this Virtual Machine be encrypted by enabling Encryption at Host?
```

WDYT?